common/array.h

```diff
@@ -221,16 +226,19 @@ class Array {
 
 	// TODO: insert, remove, ...
 
+	/** Return a pointer to the underlying memory serving as element storage. */
 	T &operator[](size_type idx) {
 		assert(idx < _size);
 		return _storage[idx];
 	}
 
+    /** Return a pointer to the underlying memory serving as element storage. */
```

Comment:
These an a few of the following look like whitespace errors

Reply:
thanks, I'll make sure to align with function declaration

Reply:
They all seem well-aligned actually, you can check it for yourself in the raw source view of the file -> https://github.com/scummvm/scummvm/blob/3bd1b124f233d2a66cecf532de1f60e6f1d6140f/common/array.h

Unless you meant something else by 'whitespace error'? The whitespace between the full stop and doxygen comment closing is intentional and used consistently everywhere as part of doxygen style, for better readability.